orderer/consensus/hlmirbft/chain.go

```diff
@@ -524,6 +524,26 @@ func (c *Chain) Consensus(req *orderer.ConsensusRequest, sender uint64) error {
 	return nil
 }
 
+//FLy2-64 - proposed changes
+// - check for forward flag in payload
+func checkForwardFlag(reqPayload []byte) bool {
+	forwardBytes := reqPayload[:8]
+	if string(forwardBytes) == ForwardFlag {
+		return true
+	}
+	return false
+}
+
+//FLY2-64 - proposed changes
+// - append forward bytes
+
+func appendForwardFlag(reqPayload []byte) []byte {
```

Comment:
Instead of appending you're prepending the flag so replace usages of the word append with prepend